internal/constants/errorclarification_test.go

```diff
@@ -0,0 +1,317 @@
+package constants
+
+import (
+	"testing"
+)
+
+func TestTranslateExitCodeToErrorClarification(t *testing.T) {
```

Comment:
If we just convert the error clarification to an int, then we don't need this test. 
We should also try to repro each condition in a test case, since this improves the overall quality of the extension.